tests/test_connections/test_abci/test_abci.py

```diff
@@ -496,10 +499,16 @@ def test_ensure_connected_raises_connection_error() -> None:
 
 
 def test_encode_varint_method() -> None:
-    """Test encode_varint method for _TendermintABCISerializer"""
-    assert _TendermintABCISerializer.encode_varint(10) == b"\x14"
-    assert _TendermintABCISerializer.encode_varint(70) == b"\x8c\x01"
-    assert _TendermintABCISerializer.encode_varint(130) == b"\x84\x02"
+    """Test encode_varint (uint64) method for _TendermintABCISerializer"""
+    assert _TendermintABCISerializer.encode_varint(1 << 31) == b"\x80\x80\x80\x80\x08"
+    assert (
+        _TendermintABCISerializer.encode_varint(1 << 63)
+        == b"\x80\x80\x80\x80\x80\x80\x80\x80\x80\x01"
+    )
+    assert (
+        _TendermintABCISerializer.encode_varint(1 << 127)
+        == b"\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x80\x02"
+    )
```

Comment:
had to change this. There is another test that covers encoding / decoding:

https://github.com/valory-xyz/open-autonomy/blob/0b885133dafeb3d170649b9f6b2dc526e0f57286/tests/test_connections/test_abci/test_abci.py#L514-L524